extensions/security/runtime/src/main/java/io/quarkus/security/runtime/QuarkusIdentityProviderManagerImpl.java

```diff
@@ -101,16 +102,16 @@ public Uni<? extends SecurityIdentity> apply(SecurityIdentity securityIdentity)
      * @return The first identity provider that was registered with this type
      */
     public SecurityIdentity authenticateBlocking(AuthenticationRequest request) {
-        List<IdentityProvider> providers = this.providers.get(request.getClass());
+        var providers = this.providers.get(request.getClass());
```

Comment:
To me it looks like it is becoming less obvious what the actual var type is, we'll have no idea what `providers` are when we look at this code in 6 months :-), I don't really mind, I'm just not sure about across the board replacement of local type references with `var`...

Reply:
I changed this because `List<IdentityProvider>` is a raw type and `var` is shorter to write :-) I don't give importance to things like these, for me it's like a book. I'll add it back (with a wildcard) because if it is better readable for someone like you, it should be there.

Reply:
@michalvavrik It is ok, you don't have to...

Reply:
NP, I am finishing other PR, so if this one is not merged in an hour, I'll change it. One way or another thanks.